src/components/Canvas/index.ts

```diff
@@ -1028,8 +1028,8 @@ export default class Canvas extends EventDispatcher {
         columnIndex,
       });
       this.emit(CanvasEvents.DATA_CHANGE, this.data);
+      this.emit(CanvasEvents.STROKE_END, this.strokedPixels, this.data);
```

Comment:
`this.strokedPixels` seems to be an array you might need to pass a nonempty array when emit the event. Maybe we could return an array in `paintSameColorRegion` and pass that to the emitter?

Reply:
You don't necessarily have to pass this.storkedPixels. You can simply pass an array to the emitter.